### PR TITLE
Fix-default-stylesheet

### PR DESCRIPTION
**Pull request by joachimhgg**
**Description**

Set default value of Stylesheet for each OS. Fix bug with linux stylesheets for the first launch.

clean your preferences before to test:

`python3 scripts/preferences_cleaner/preferences_cleaner.py`

### CODE REVIEW

File: src/webots/gui/WbGuiApplication.cpp [hunk 1]
```diff
@@ -611,13 +611,8 @@ static void setDarkTitlebar(HWND hwnd) {
...
 void WbGuiApplication::udpateStyleSheet() {
-#ifdef __linux__
-  const QString themeToLoad = WbPreferences::instance()->value("General/theme", "webots_night.qss").toString();
-#else
-  const QString themeToLoad = WbPreferences::instance()->value("General/theme", "webots_classic.qss").toString();
-#endif
-
-  QFile qssFile(WbStandardPaths::resourcesPath() + themeToLoad);
+  mThemeLoaded = WbPreferences::instance()->value("General/theme").toString();
+  QFile qssFile(WbStandardPaths::resourcesPath() + mThemeLoaded);
```
Comment (BenjaminDeleze) [inline src/webots/gui/WbGuiApplication.cpp:615]: Is it normal that we never close the `qssFile`?

**Review by BenjaminDeleze — COMMENTED** [review]
Seems good, just a question.

File: src/webots/gui/WbGuiApplication.cpp (hunk 1 above)
Comment (joachimhgg) [reply src/webots/gui/WbGuiApplication.cpp:615]: `QFile` object is closed automatically upon its destructor, hence we do not need to call `close()` explicitly.

**Review by BenjaminDeleze — APPROVED** [review]

**Comment by BenjaminDeleze** [comment]
Thank you

**Pull request merged**